frontend/src/components/FilterDrawer.vue

```diff
@@ -1,5 +1,20 @@
 <template>
   <v-sheet class="mx-auto" elevation="8">
+    <v-card order="1">
+      <!-- <v-card-title>Search Text Within:</v-card-title> -->
+      <v-card-text>
+        <!-- <v-btn-toggle v-model="textSearchFields" @update:modelValue="filter" class="mb-2" multiple outlined
+          variant="text" divided>
+          <v-btn value="long_name">Title</v-btn>
+          <v-btn value="citation">Citation</v-btn>
+          <v-btn value="textmodel_snipped">Abstract</v-btn>
+        </v-btn-toggle> -->
```

Comment:
I take it we are okay searching across all fields?
If yes, remove the commented template items.

Reply:
But if you want this to actually function to search models, you need to also modify all of the associated functions, for example here:
https://github.com/CUAHSI/hydroprocess-db/blob/search_improvements/frontend/src/components/FilterDrawer.vue#L186-L204